### PR TITLE
Add PostgreSQL operator with deps

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: server/filters.go [hunk 1]
```diff
@@ -132,6 +132,75 @@ func pxcFilter(versions map[string]*pbVersion.Version, apply string, current str
...
+	sorted, err := sortedVersionsDesc(keys)
+	if err != nil {
+		return status.Errorf(codes.Internal, "failed to sort versions: %v", err)
+	}
+
+	if (strings.ToLower(apply) == recommended || strings.ToLower(apply) == latest) && current == "" {
+		return deleteOtherBut(sorted[0].String(), versions)
+	}
+
+	desired := apply //assume version number
```
Comment (qJkee) [inline server/filters.go:158]: can we move this to beginning with
`desured := strings.ToLower(apply)`
coz at this place we call `ToLower` func 3 times already, and then 2 times more on the same value

**Review by nonemax — APPROVED** [review]

File: server/filters.go (hunk 1 above)
Comment (blez) [reply server/filters.go:160]: https://github.com/Percona-Lab/percona-version-service/pull/37/commits/61d255af3021eeea2d2912702e9edcaf779059df

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by qJkee — APPROVED** [review]

**Pull request merged**